hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/SnapshotScannerHDFSAclHelper.java

```diff
@@ -474,8 +474,8 @@ void createTableDirectories(TableName tableName) throws IOException {
    */
   List<Path> getTableRootPaths(TableName tableName, boolean includeSnapshotPath)
       throws IOException {
-    List<Path> paths = Lists.newArrayList(pathHelper.getTmpTableDir(tableName),
```

Comment:
Found out this was creating table tmp dir on CreateTableProcedure.CREATE_TABLE_POST_OPERATION when ACL is enabled,